### ISSUE TITLE
Adding an express server as backend

### ISSUE THREAD

**Issue by flavienrako** [Avaray/yet-another-neutralinojs-starter#1]
Thank you for your repo, a great job, is there a way to support an express server?

**Comment by Avaray** [comment]
> Thank you for your repo, a great job, is there a way to support an express server?

Yes, but you need to have JS runtime like Node/Bun/Deno to run express.
If devices will have one of them installed then you just spawn a process.
Otherwise you can bundle a runtime, and then spawn a process.
